packages/amplify-cli/src/cli.js

```diff
@@ -28,6 +28,8 @@ async function run(argv) {
 
   const context = await cli.run(argv);
 
+  context.ignoreSSL = !(argv.includes('--ignoressl') || argv.includes('--ignore-ssl'));
+
```

Comment:
This won't work, the cli.run(argv) won't carries out the actual execution of the command logic. 
Adding something to the context right before the return won't have any effect on the command execution. 
And why is there a `!` ?

Reply:
Wouldn't this grab the flag from the cli and set it into the context? As for the !, I use that to translate between the **true** the user states and the **false** required for rejectUnauthorized.

Make it easier for the user to set the options.

Reply:
if you want to access the input options such as '--ignoressl' in your code, try to use
`context.parameters.options`

again, please test your implementation and make sure it works end-to-end.